C#/v1/NoarkWsClientSample/NoarkWsClientSample/FinalizationSample.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Documaster.WebApi.Client.Noark5;
+using Documaster.WebApi.Client.Noark5.Client;
+using Documaster.WebApi.Client.Noark5.NoarkEntities;
+
+namespace NoarkWsClientSample
+{
+    public class FinalizationSample
+    {
+        private readonly DocumasterClients documasterClients;
+
+        public FinalizationSample(DocumasterClients documasterClients)
+        {
+            this.documasterClients = documasterClients;
+        }
+
+        public void FinalizeSeries(string seriesId)
+        {
+            NoarkClient client = this.documasterClients.GetNoarkClient();
+
+            // Finalize series by setting changing the series status to Closed Period (P).
+            // Finalizing the series will not finalize child objects!
+
+            Arkivdel series = GetNoarkEntityById<Arkivdel>(seriesId);
+            series.Arkivdelstatus = Arkivdelstatus.AVSLUTTET_PERIODE;
+
+            client.Transaction()
```

Comment:
Ideally, the Finalize methods would just add Save operations on a provided transaction object and we would call .Commit() when all operations are added. That way we could finalize all the objects in the same transaction. Or none if an error occurrs.

Reply:
See new sample code.